tests/State/ScopeTest.php

```diff
@@ -49,6 +49,29 @@ public function testSetTags(): void
         $this->assertSame(['foo' => 'bar', 'bar' => 'baz'], $event->getTags());
     }
 
+    public function testRemoveTag(): void
+    {
+        $scope = new Scope();
+        $event = $scope->applyToEvent(Event::createEvent());
+
+        $this->assertNotNull($event);
+        $this->assertEmpty($event->getTags());
```

Comment:
There is no need for these lines as the "base" case is that a scope has some tag and they get removed, we don't care to assert that the scope is initially empty

Reply:
Thanks, I've removed these 2 lines. Please let me know if you would further cleanup the case from unnecessary stuff.

Reply:
I think that it's fine, the only thing I forgot to write is that since the newly added method is removing a specific tag it would be better to add to the scope two tags so that we can assert that only the right one is removed